country-a-service/epps-service/src/main/java/dk/sundhedsdatastyrelsen/ncpeh/service/VaccinationService.java

```diff
@@ -0,0 +1,29 @@
+package dk.sundhedsdatastyrelsen.ncpeh.service;
+
+import dk.nsp.test.idp.EmployeeIdentities;
+import dk.sundhedsdatastyrelsen.ncpeh.client.DdvClient;
+import dk.vaccinationsregister.schemas._2013._12._01.GetVaccinationCardRequestType;
+import dk.vaccinationsregister.schemas._2013._12._01.VaccinationType;
+import jakarta.xml.bind.JAXBException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class VaccinationService {
```

Comment:
I'm afraid that we are in danger of repeating the OpenNCP FHIR issue: Enabling for everybody a new feature that no one needs.

Should we hide the PS-services behind a "feature flag" until we start integration for real-real?  I think we could use `@ConditionalOnProperty` for this.

Reply:
Or maybe the simpler solution is:  Don't annotate with `@Service` before we need it.  As far as I can tell, the integration test does not use the Spring bean anyway.